test/test_document_store.py

```diff
@@ -465,13 +496,21 @@ def test_delete_documents_by_id_with_filters(document_store_with_docs):
     assert all(doc.id in all_ids_left for doc in docs_not_to_delete)
 
     
-def test_delete_documents_with_filters(document_store_with_docs):
-    document_store_with_docs.delete_documents(filters={"meta_field": ["test1", "test2"]})
-    documents = document_store_with_docs.get_all_documents()
-    assert len(documents) == 1
-    assert documents[0].meta["meta_field"] == "test3"
+def test_delete_documents_by_id(document_store_with_docs):
```

Comment:
Maybe I am missing something here but there's still another test with the same name in line 473ff

Reply:
You're right. I merged into one test now. They were essentially the same.